go-controller/pkg/util/ovs.go

```diff
@@ -455,8 +417,8 @@ func RunOVSDBTool(args ...string) (string, string, error) {
 	return strings.Trim(strings.TrimSpace(stdout.String()), "\""), stderr.String(), err
 }
 
-// RunOVSDBClientOVN runs an 'ovsdb-client [OPTIONS] COMMAND [SERVER] [ARG...] command' against OVN NB database.
-func RunOVSDBClientOVNNB(command string, args ...string) (string, string, error) {
+// runOVSDBClientOVN runs an 'ovsdb-client [OPTIONS] COMMAND [SERVER] [ARG...] command' against OVN NB database.
```

Comment:
this doc string is wrong missing "NB". You aren't removing the function here as stated in the commit message, you are just making it not exported. Why?

Reply:
Fixed.